dashboard/app/views/projects_lists/index.html.haml

```diff
@@ -0,0 +1,13 @@
+- content_for :head do
+  %script{src:'/assets/js/projectsList.js'}
+
+#projects-list
+
+:ruby
+  reactProps = {
+    projectsData: @projects_list_data
+  }
+
+:javascript
+  ReactDOM.render(React.createElement(window.dashboard.ProjectsList, #{reactProps.to_json}),
+    document.getElementById('projects-list'));
```

Comment:
👍 thanks for pointing this out, I like the data attributes approach much better!